common/lib/protocol-definitions/api-report/protocol-definitions.legacy.alpha.api.md

```diff
@@ -1,478 +0,0 @@
-## Alpha API Report File for "@fluidframework/protocol-definitions"
```

Comment:
Note for reviewers: this package was already generating `legacy.beta` and `legacy.alpha` reports. This is just removing the redundant `legacy.alpha` one.

Reply:
Looks like it has been built failed under this directory, which skip the `legacy.public` generation. Rebuild this package and it shows up.